### ISSUE TITLE
Programs page timeline doesn't work on mobile

### ISSUE THREAD

**Issue by vj-codes** [anitab-org/anitab-org.github.io#333]
### Describe the bug
On your mobile, click on the deployed website and go to programs section. 
The timeline is broken and doesn't work as expected. 

### Expected behavior

The timeline should be responsive and adjusted as per the screen, ideally displayed vertically instead of horizontally. 

### Screenshots

https://user-images.githubusercontent.com/60894542/139511803-2181e19b-bb0b-4187-9928-546d6bdef8e8.mp4

### Smartphone (please complete the following information):

- Device: Redmi Note 10 pro max
- OS: Android 11
- Browser: Chrome

**Comment by abritpal** [comment]
Should I take this up?

**Comment by vj-codes** [comment]
Assigning you @abritpal 
Happy coding!

**Comment by abritpal** [comment]
> The timeline should be responsive and adjusted as per the screen, ideally displayed vertically instead of horizontally.
@vj-codes, what do you mean by vertically here, should it scroll vertically or the title of the program and timeline should be adjusted vertically, as in displayed verticaly.

**Comment by vj-codes** [comment]
@abritpal yes it should be displayed vertically with program names and respective details

**Comment by nandini45** [comment]
@abritpal  are you working on it still??

**Comment by Jyotika2001** [comment]
Has this issue been solved? If not, I would like to work on it.

**Comment by nandini45** [comment]
@Jyotika2001 sure! go ahead
 please keep updating us with your progress.

**Comment by nandini45** [comment]
@abritpal  you have not given any updates and it has been around two months
i am unassigning you the issue

**Comment by Jyotika2001** [comment]
Hey @nandini45, I was going through the website on my mobile. Many changes have been made to the website layout from the time this issue was raised for example the 'programs' section is called the 'events and programs' now. My doubt was where exactly  is this timeline that I need to work on? The 'events and programs' section now consists of 3 sub-headings-People we serve, events and programs and awards & grants, none of which consist of a timeline.

**Comment by nandini45** [comment]
@Jyotika2001  let me check 
i will get back to you by the end of the day

**Comment by Jyotika2001** [comment]
@nandini45 Please could you tell me the page so that I can start working.

**Comment by nandini45** [comment]
@Jyotika2001  idk which link u are opening but when i  am trying to access it on my device it is broken and showing the same behavior as explained 
see as vaishnavi said the behavior should be the timeline should be responsive and adjusted as per the screen, ideally displayed vertically instead of horizontally.
https://anitab-org.github.io/ this is  the link
(in the program section the timeline is given)
recheck again please
@vj-codes  can you please look into it and confirm the issue

**Comment by Yash-271120** [comment]
@vj-codes @nandini45 This issue seems to be not fixed yet. Can I work on this?
